main.go

```diff
@@ -91,6 +91,11 @@ func doMain(args []string) {
 					Name:  "tenant_id",
 					Usage: "Tenant ID (optional) - propagate for just a single tenant.",
 				},
+				cli.DurationFlag{
+					Name:  "throttle",
+					Usage: "`DURATION` to wait for each reindexing batch request",
```

Comment:
please sespecifycify what is a DURATION with units.

Reply:
also I find is slightly confusing name of the option is throttle and the usage says duration.

Reply:
I have not checked but will we have some examples of the command in the help? and: do we need to update the docs.mender.io?

Reply:
Changed the flat to `--rate-limit`. Should be more self-explanatory.